src/components/play/PlayView.tsx

```diff
@@ -255,6 +256,15 @@ const PlayView = () => {
           }
         })()}
       </Grid>
+
+      <PlayHints
+        hints={currentQuestion.data.hints}
+        isCorrect={isCorrect}
+        currentAttempts={userAnswers.length}
+        maxAttempts={maxAttempts}
+        mt={4}
```

Comment:
I think this can stay in the component, no need for a prop to set the margin ...